### ISSUE TITLE
Add LocalizationPart to everything

### ISSUE THREAD

**Issue by MatteoPiovanelli** [OrchardCMS/Orchard#7914]
the comments from #7911 got me thinking
I would like to have a Command from Orchard.Localization to add LocalizationPart to every ContentType, and a recipe available that calls it (so I can really call it from the backend rather than from a command line). I would not have this run automatically when Orchard.Localization is enabled.

We should discuss whether there are specific ContentTypes where adding a LocalizationPart would be an issue, so that we can filter them out in the Command.
I call for @sebastienros and @HermesSbicego-Laser to pitch in on this, and to add anyone they think may be able to contribute to this discusion.
After finding those types, the implementation of this should be trivial

**Comment by sebastienros** [comment]
We can just have a Recipe that would enable Localization and add the part to common types, but not every single of them.

**Comment by MatteoPiovanelli** [comment]
Yes we can. I wonder if that is the way that leads to less work down the line. I know that wouldn't be the case for us, because we rarely (if ever) use "default" content types (e.g. Page). I'd rather see if we can find a "thing" that all types that are generally not localized share.
I am not saying that the recipe you propose is not the right way, or the way we should be implementing this, and honestly I can't think of any discriminator that would work and tell me whether a given type should have a LocalizationPart.

**Comment by sebastienros** [comment]
I prefer these solutions in this order:
1- Add LocalizationPart to Blog and Page only
2- Don't add LocalizationPart to anything

And I very much dislike adding it to everything.

If you chose 2, then there is nothing to do. If you chose 1, then we need a custom recipe to enable Localization with some default settings.

**Comment by MatteoPiovanelli** [comment]
Out of those, my preferred solution is 2, for totally egoistic reasons (we hardly ever use Page in our tenants).
Question: is it possible to have "conditions" in recipes? As in, have the recipe add the LocalizationPart to Page only if the Page ContentType exists in the tenant already. I have never seen this myself, but maybe it's a feature that's just never come up in my time with Orchard.
